src/ppl/README.md

```diff
@@ -154,78 +95,208 @@ pin placement grid defined with `define_pin_shape_pattern` use:
 
+```tcl
+set_pin_thick_multiplier 
+    [-hor_multiplier h_mult]
+    [-ver_multiplier v_mult]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-hor_multiplier` | Defines the thickness multiplier for the horizontal pins. |
+| `-ver_multiplier` | Defines the thickness multiplier for the vertical pins. |
+
+### Set Simulated Annealing Parameters
+
+The `set_simulated_annealing` command defines the parameters for simulated annealing pin placement.
+
+```tcl
+set_simulated_annealing
+    [-temperature temperature]
+    [-max_iterations iter]
+    [-perturb_per_iter perturbs]
+    [-alpha alpha]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-temperature` | Specify the temperature parameter, must be positive. |
+| `-max_iterations` | Specify the maximum number of iterations. |
+| `-perturb_per_iter` | Specify the number of perturbations per iteration. |
+| `-alpha` | Specify the temperature decay factor, must be positive. |
+
+### Place Individual Pin
+
+The `place_pin` command places a specific pin in the specified location, with the specified size.
+It is recommended that the placement of individual pins be done before the `place_pins` command,
+as the routing tracks occupied by these individual pins will be blocked, preventing overlaps.
+
+To place an individual pin:
+
+```tcl
+place_pin 
+    [-pin_name pin_name]
+    [-layer layer]
+    [-location {x y}]
+    [-pin_size {width height}]
+    [-force_to_die_boundary]
 ```
-set_pin_thick_multiplier [-hor_multiplier h_mult]
-                         [-ver_multiplier v_mult]
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-pin_name` | Option is the name of a pin of the design. |
+| `-layer` | Defines the routing layer where the pin is placed. |
+| `-location` | Defines the center of the pin (in microns). |
+| `-pin_size` | Option defines the width and height of the pin (in microns). |
+| `-force_to_die_boundary` | When this flag is enabled, the pin will be snapped to the nearest routing track, next to the die boundary. |
+
+### Place All Pins
+
+Use the following command to perform pin placement:
+
+```tcl
+place_pins 
+    [-hor_layers h_layers]
+    [-ver_layers v_layers]
+    [-random_seed seed]
+    [-random]
+    [-corner_avoidance length]
+    [-min_distance distance]
+    [-min_distance_in_tracks]
+    [-exclude region]
+    [-group_pins pin_list]
+    [-annealing]
 ```
 
--   The `-hor_multiplier` option defines the thickness multiplier for the horizontal pins.
--   The `-ver_multiplier` option defines the thickness multiplier for the vertical pins.
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-hor_layers` | Specify the layers to create the metal shapes of pins placed in horizontal tracks. Can be a single layer or a list of layer names. |
+| `-ver_layers` | Specify the layers to create the metal shapes of pins placed in vertical tracks. Can be a single layer or a list of layer names. |
```

Comment:
```suggestion
| `-hor_layers` | Specify the layers to create the metal shapes of pins placed in horizontal tracks. It can be a single layer or a list of layer names. |
| `-ver_layers` | Specify the layers to create the metal shapes of pins placed in vertical tracks. It can be a single layer or a list of layer names. |
```